src/app.jsx

```diff
@@ -144,3 +138,70 @@ export class Application extends React.Component {
         );
     }
 }
+
+const DropdownWithKebab = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const onToggleClick = () => {
+        setIsOpen(!isOpen);
+    };
+    const onSelect = (_event, itemId) => {
+        setIsOpen(false);
+    };
+    return (
+        <Dropdown
+            isPlain
+            isOpen={isOpen}
+            onSelect={onSelect}
+            onOpenChange={isOpen => setIsOpen(isOpen)}
+            toggle={toggleRef =>
+                <MenuToggle ref={toggleRef} aria-label="kebab dropdown toggle" variant="plain" onClick={onToggleClick} isExpanded={isOpen}>
+                    <EllipsisVIcon />
+                </MenuToggle>}
+        >
+            <DropdownList>
+                <DropdownItem itemId={0} key="action1">
+                    Action
+                </DropdownItem>
+                <DropdownItem itemId={1} key="action2">
+                    Action
+                </DropdownItem>
+                <Divider component="li" key="separator" />
+                <DropdownItem itemId={2} key="action3">
+                    Action
+                </DropdownItem>
+            </DropdownList>
+        </Dropdown>
+    );
+};
+
+export const DropdownBasic = () => {
+    const [isOpen, setIsOpen] = useState(false);
+    const onToggleClick = () => {
+        setIsOpen(!isOpen);
+    };
+    const onSelect = (_event, itemId) => {
+        console.log('selected', itemId);
+        setIsOpen(false);
+    };
+    return (
+        <Dropdown
+            isOpen={isOpen}
+            onSelect={onSelect}
+            onOpenChange={isOpen => setIsOpen(isOpen)}
+            toggle={toggleRef => <MenuToggle ref={toggleRef} onClick={onToggleClick} isExpanded={isOpen} variant="secondary" splitButtonOptions={{ variant: "action", items: [<Icon key="list-icon"><ListIcon /></Icon>] }} />} ouiaId="BasicDropdown"
```

Comment:
PLease removed the ouiaId, these are used for some testing in patternfly but we don't use it.